content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,314 @@
-   impossible to provide product support. **DO NOT** change the log driver of an installation to anything other than `json-file` or `journald`.
+```
+tfe-admin app-config -k log_forwarding_enabled -v 1
+```
 
-## Audit Logs
+When enabled, the Terraform Enterprise application settings should show the
+following for the `log_forwarding_enabled` setting:
+
+```
+    ...
+    "log_forwarding_enabled": {
+        "value": "1"
+    },
+    ...
+```
+
+Log forwarding is disabled by default.
+
+### Configure External Destinations
+
+Once log forwarding is enabled, it must be configured to forward logs to one or
+more external destinations. To configure log forwarding, the
+`log_forwarding_config` Terraform Enterprise application setting must contain
+valid [Fluent Bit `[OUTPUT]` configuration](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit)
+specifying which external destination(s) to forward logs to. Since the Terraform
+Enterprise application settings are stored as JSON strings, it is recommended to
+create a `fluent-bit.conf` file with the valid Fluent Bit `[OUTPUT]`
+configuration and then configure the `log_forwarding_config` Terraform
+Enterprise application setting with the contents of that `fluent-bit.conf` file.
+That way, the configuration is stored in the Terraform Enterprise application
+settings exactly how it appears in the `fluent-bit.conf` file.
+
+Configure log forwarding for a Standalone installation of Terraform Enterprise:
+
+```
+replicatedctl app-config set log_forwarding_config --value "$(cat fluent-bit.conf)"
+```
+
+Configure log forwarding for an Active/Active installation of Terraform
+Enterprise:
+
+```
+tfe-admin app-config -k log_forwarding_config -v "$(cat fluent-bit.conf)"
+```
+
+Once configured, the Terraform Enterprise application settings should show the
+`log_forwarding_config` setting in escaped JSON string format:
+
+```
+    ...
+    "log_forwarding_config": {
+        "value": "# Match all logs and do not forward them anywhere.\n[OUTPUT]\n    Name null\n    Match *\n"
+    },
+    ...
+```
+
+The default configuration does not forward any logs.
+
+```
+# Match all logs and do not forward them anywhere.
+[OUTPUT]
+    Name null
+    Match *
+```
+
+To forward logs to multiple external destinations, use multiple `[OUTPUT]`
+directives.
+
+```
+# Forward all logs to Datadog.
+[OUTPUT]
+    Name datadog
+    Match *
+    ...
+
+# Forward all logs to Fluent Bit or Fluentd.
+[OUTPUT]
+    Name forward
+    Match *
+    ...
+```
+
+~> **NOTE:** Do not use an `[OUTPUT]` directive with the
+[`stdout` Fluent Bit output plugin](https://docs.fluentbit.io/manual/pipeline/outputs/standard-output).
+Doing so will result in a loop that will continuously emit logs!
+
+## Supported External Destinations
+
+These are the supported external destinations one can forward logs to. Each
+supported external destination contains example configuration for convenience.
```

Comment:
I wonder if we need to have a note or callout or something in the section about how to set up external forwarding (maybe in the first paragraph) to be like 'You have to pick one of the ones below..." or somehow call this out?

```suggestion
You can only forward logs to one of the supported external destinations below. Each
supported external destination contains example configuration for convenience.
```